tests/test_homo_sapiens.py

```diff
@@ -53,6 +53,12 @@ def test_chromosome_lengths(self):
         self.assertEqual(genome.chromosomes["chrX"].length, 156040895)
         self.assertEqual(genome.chromosomes["chrY"].length, 57227415)
 
+    def test_chromosome_recombination_maps(self):
+        for chrom in homo_sapiens.genome.chromosomes:
+            chrom = homo_sapiens.genome.chromosomes[chrom]
+            cm = chrom.recombination_map()
+            assert(type(cm) == msprime.simulations.RecombinationMap)
```

Comment:
We're using unittest here, so we need to use the built int assert methods, e.g.
```
self.assertIsInstance(cm, msprime.RecombinationMap)
```

Also, we need to test the error cases here. What happens when you don't know the key?

Reply:
got it, will finish after I make a few changes discussed below

Reply:
so here are you talking about what we should do when we don't know the key of a specific chromosome? like if a user were to do
`chrom = homo_sapiens.genome.chromosomes["blahblah"]`

Reply:
the code should quit with a KeyError